docs/workflow/blocks/form.rst

```diff
@@ -80,64 +80,4 @@ A simple search form without a submit button.
         }
     },
     "uiSchema": {}
-  }
```

Comment:
I didn't get the purpose of removing this yet and just wanted to check.

Reply:
I'll bring the complex example back. I was just removing all the updates added at the time of the break.